ajax/libs/lave/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "lave",
+  "filename": "lave.min.js",
+  "version": "1.1.9",
+  "description": "eval in reverse: stringifying all the stuff that JSON.stringify won't",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jed/lave.git"
+  },
+  "keywords": [
+    "stringify",
+    "uneval",
+    "circular",
+    "json",
+    "serialize"
+  ],
+  "license": "MIT",
+  "npmName": "lave",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```

Comment:
There are 6 files in latest npm ackage:
`index.js  lave.js  LICENSE  package.json  README.md  test.js`

So why to use `"**/*"` here? We don't need them all. Why not just `"lave*"`?